features/step_definitions/assertions.rb

```diff
@@ -6,67 +6,77 @@
 
 Then(/^git wrapper permissions are 0700$/) do
   permissions_test = %Q([ $(stat -c "%a" #{TestApp.git_wrapper_path_glob}) == "700" ])
-  _stdout, _stderr, status = vagrant_cli_command("ssh -c #{permissions_test.shellescape}")
-
-  expect(status).to be_success
+  expect { run_remote_ssh_command(permissions_test) }.not_to raise_error
 end
 
 Then(/^the shared path is created$/) do
-  run_vagrant_command(test_dir_exists(TestApp.shared_path))
+  run_remote_ssh_command(test_dir_exists(TestApp.shared_path))
 end
 
 Then(/^the releases path is created$/) do
-  run_vagrant_command(test_dir_exists(TestApp.releases_path))
+  run_remote_ssh_command(test_dir_exists(TestApp.releases_path))
 end
 
 Then(/^(\d+) valid releases are kept/) do |num|
   test = %Q([ $(ls -g #{TestApp.releases_path} | grep -E '[0-9]{14}' | wc -l) == "#{num}" ])
-  _, _, status = vagrant_cli_command("ssh -c #{test.shellescape}")
-  expect(status).to be_success
+  expect { run_remote_ssh_command(test) }.not_to raise_error
 end
 
 Then(/^the invalid (.+) release is ignored$/) do |filename|
   test = "ls -g #{TestApp.releases_path} | grep #{filename}"
-  _, _, status = vagrant_cli_command("ssh -c #{test.shellescape}")
-  expect(status).to be_success
+  expect { run_remote_ssh_command(test) }.not_to raise_error
 end
 
 Then(/^directories in :linked_dirs are created in shared$/) do
   TestApp.linked_dirs.each do |dir|
-    run_vagrant_command(test_dir_exists(TestApp.shared_path.join(dir)))
+    run_remote_ssh_command(test_dir_exists(TestApp.shared_path.join(dir)))
   end
 end
 
 Then(/^directories referenced in :linked_files are created in shared$/) do
   dirs = TestApp.linked_files.map { |path| TestApp.shared_path.join(path).dirname }
   dirs.each do |dir|
-    run_vagrant_command(test_dir_exists(dir))
+    run_remote_ssh_command(test_dir_exists(dir))
   end
 end
 
 Then(/^the repo is cloned$/) do
-  run_vagrant_command(test_dir_exists(TestApp.repo_path))
+  run_remote_ssh_command(test_dir_exists(TestApp.repo_path))
 end
 
 Then(/^the release is created$/) do
-  run_vagrant_command("ls -g #{TestApp.releases_path}")
+  run_remote_ssh_command("ls -g #{TestApp.releases_path}")
+end
+
+Then(/^the REVISION file is created in the release$/) do
+  stdout, _stderr = run_remote_ssh_command("cat #{@release_paths[0]}/REVISION")
+
+  # TODO: confirm that this value is actually constant for everyone
+  expect(stdout.strip).to eq "3f41365139a0b3a408e96f13c781d9cedba8b4b8"
```

Comment:
I think it would be better to match this with a regular expression, because I doubt this value is going to be stable. How about:

```suggestion
  expect(stdout.strip).to match(/\h{40}/)
```